docs/CONTRIBUTING-id.md

```diff
@@ -16,60 +16,60 @@ Dengan berkontribusi, Anda setuju untuk menghormati [Kode Etik](CODE_OF_CONDUCT-
 <!----><a id="in-a-nutshell"></a>
 ## Versi pendek
 
-1. "Tautan untuk mengunduh buku dengan mudah" tidak selalu merujuk pada buku yang *gratis*. Harap hanya memberikan konten yang gratis. Pastikan itu benar-benar gratis. Kami tidak menerima tautan ke halaman yang *meminta* alamat email yang valid untuk mengunduh buku, tetapi kami menyambut daftar situs yang meminta alamat email.
+1. "Tautan download buku" tidak selalu merujuk pada buku yang benar-benar *gratis*. Mohon untuk hanya mendaftarkan tautan ke buku/konten yang benar-benar gratis. Kami tidak menerima tautan ke halaman website yang *membutuhkan* alamat email dari pengguna sebelum mengunduh atau mengakses kontennya. Kami bisa menerima tautan yang membutuhkan alamat email pengguna (atau yang serupa) jika pada bagian keterangan sesuai dengan panduan yang kami berikan.
```

Comment:
The changes I would suggest are:
- The english word "download" and mixed phrase "halaman website" should be reverted and replaced back to the indonesian word "mengunduh" and "situs web" for consistency.

Since the last sentence is an expansion of the original sentence in the English version of this file, I would like to also request some changes while I'm here 😄 
- The word "bisa" should be changed to "hanya dapat" to be more standardized and concrete.
- The word "pada" should be deleted as it makes the sentence an ineffective sentence.

```suggestion
1. "Tautan untuk mengunduh buku" tidak selalu merujuk pada buku yang benar-benar *gratis*. Mohon untuk hanya mendaftarkan tautan ke buku/konten yang benar-benar gratis. Kami tidak menerima tautan ke situs web yang *membutuhkan* alamat email dari pengguna sebelum mengunduh atau mengakses kontennya. Kami hanya dapat menerima tautan yang membutuhkan alamat email pengguna (atau yang serupa) jika bagian keterangan sesuai dengan panduan yang kami berikan.
```